File: Dashboard.Models/PackageNote.cs
using Newtonsoft.Json;

namespace Dashboard.Models
{
    public class PackageNote
    {
        [JsonIgnore]
        public int PackageNoteId { get; set; }

        public string Packaging { get; set; }
        public string StagingArea { get; set; }
        public string Packer { get; set; }

        [JsonIgnore]
        public DeliveryOrder DeliveryOrder { get; set; }
    }
}

File: Dashboard/ClientApp/src/app/models/package-note.ts
export class PackageNote {
  packageNoteId: number;
  packaging: string;
  stagingArea: string;
  packer: string;

  static fromJson(packageNote: any): PackageNote {
    return {
      packageNoteId: packageNote.PackageNoteId || packageNote.packageNoteId,
      packaging: packageNote.Packaging || packageNote.packaging,
      stagingArea: packageNote.StagingArea || packageNote.stagingArea,
      packer: packageNote.Packer || packageNote.packer
    }
  }
}